<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-14E9C6J" failures="1" tests="7" name="test1" time="43.83" errors="0" timestamp="2025-05-12T12:17:39 IST">
  <testcase classname="testNgProject.BasicTestcase" name="verifyShowmessageButtonText" time="0.22"/>
  <testcase classname="testNgProject.DataProviderEx" name="verifySearch" time="3.821"/>
  <testcase classname="testNgProject.BasicTestcase" name="verifyShowmessageButtonText" time="0.173"/>
  <testcase classname="testNgProject.BasicTestcase" name="verifyTitle" time="0.041">
    <failure type="java.lang.AssertionError" message="Failed due to existing bug expected [Anagha] but found [Obsqura Testing]">
      <![CDATA[java.lang.AssertionError: Failed due to existing bug expected [Anagha] but found [Obsqura Testing]
at testNgProject.BasicTestcase.verifyTitle(BasicTestcase.java:22)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <testcase classname="testNgProject.BasicTestcase" name="verifyShowmessageButtonEnabled" time="0.0">
    <skipped/>
  </testcase> <!-- verifyShowmessageButtonEnabled -->
  <testcase classname="testNgProject.DataProviderEx" name="verifySearch" time="4.415"/>
  <testcase classname="testNgProject.DataProviderEx" name="verifySearch" time="4.285"/>
  <testcase classname="testNgProject.BasicTestcase" name="verifyShowmessageBackgroundColor" time="0.0">
    <ignored/>
  </testcase> <!-- verifyShowmessageBackgroundColor -->
</testsuite> <!-- test1 -->
